scipy/interpolate/_bsplines.py

```diff
@@ -561,6 +561,87 @@ def integrate(self, a, b, extrapolate=None):
         integral *= sign
         return integral.reshape(ca.shape[1:])
 
+    def design_matrix(self, x, kind="dense", check_finite=True):
+        '''
+        Returns a design matrix for BSpline object in "dense" "CSR" formats.
+        
```

Comment:
needs `.. versionadded:: 1.8.0` directive (let's be optimistic). Look through the codebase to see where in the docstrings this one is placed.